editor/src/templates/editor-canvas.tsx

```diff
@@ -1221,23 +1224,47 @@ export class EditorCanvas extends React.Component<EditorCanvasProps> {
         ) {
           const dragStart = this.props.editor.canvas.interactionSession.interactionData.dragStart
 
-          const newDrag = roundPointForScale(
-            Utils.offsetPoint(canvasPositions.canvasPositionRounded, Utils.negate(dragStart)),
-            this.props.model.scale,
-          )
-          this.handleEvent({
-            ...canvasPositions,
-            event: 'MOVE',
-            modifiers: Modifier.modifiersForEvent(event),
-            cursor: null,
-            nativeEvent: event,
-            interactionSession: updateInteractionViaMouse(
-              this.props.editor.canvas.interactionSession,
-              newDrag,
-              Modifier.modifiersForEvent(event),
-              null,
-            ),
-          })
+          if (document.pointerLockElement) {
```

Comment:
what is document.pointerLockElement? is it a boolean?

Reply:
oops, fixed